portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,18 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+// Automatic Slideshow - change image every 3 seconds
+var myIndex = 0;
```

Comment:
We generally try to avoid using global variables where possible, it's really easy to make complicated and hard to debug interactions with them, so it's good style to avoid them.
Maybe try instead giving carousel an argument and bind that argument in setTimeout.

Reply:
fixed - passing an integer to the function